File: Service_eHospital_DongNai_A/Implement/ThuPhanUngThuocCoHai_ThuocGayNghiNgoADRService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class ThuPhanUngThuocCoHai_ThuocGayNghiNgoADRService : BaseService<ThuPhanUngThuocCoHai_ThuocGayNghiNgoADR, IThuPhanUngThuocCoHai_ThuocGayNghiNgoADRRepository>
    , IThuPhanUngThuocCoHai_ThuocGayNghiNgoADRService
    {
    private readonly IThuPhanUngThuocCoHai_ThuocGayNghiNgoADRRepository _ThuPhanUngThuocCoHai_ThuocGayNghiNgoADRRepository;
    public ThuPhanUngThuocCoHai_ThuocGayNghiNgoADRService(IThuPhanUngThuocCoHai_ThuocGayNghiNgoADRRepository ThuPhanUngThuocCoHai_ThuocGayNghiNgoADRRepository) : base(ThuPhanUngThuocCoHai_ThuocGayNghiNgoADRRepository)
    {
    _ThuPhanUngThuocCoHai_ThuocGayNghiNgoADRRepository = ThuPhanUngThuocCoHai_ThuocGayNghiNgoADRRepository;
    }
    }
    }

File: CRM/src/app/shared/eHospital_DongNai_A/ThuPhanUngThuocCoHai_ThuocGayNghiNgoADR.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { ThuPhanUngThuocCoHai_ThuocGayNghiNgoADR } from './ThuPhanUngThuocCoHai_ThuocGayNghiNgoADR.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class ThuPhanUngThuocCoHai_ThuocGayNghiNgoADRService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'ThuocGayNghiNgoADR_Id', 'PhanUngThuocCoHai_Id', 'Duoc_Id', 'HamLuong', 'LieuDungMotLan', 'SoLanDung', 'DuongDung_Id', 'NgayBatDau', 'NgayKetThuc', 'HanSuDung', 'HangSanXuat_Id', 'NguoiTao_Id', 'NgayTao', 'NguoiCapNhat_Id', 'NgayCapNhat', 'LyDoDungThuoc'];

    List: ThuPhanUngThuocCoHai_ThuocGayNghiNgoADR[] | undefined;
    ListFilter: ThuPhanUngThuocCoHai_ThuocGayNghiNgoADR[] | undefined;
    FormData!: ThuPhanUngThuocCoHai_ThuocGayNghiNgoADR;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "ThuPhanUngThuocCoHai_ThuocGayNghiNgoADR";
    }
}

